racker/salus-telemetry-presence-monitor
# Use default jib base image, which is now non-Alpine anyway

# What

The code with built with Java 11 now, so this app's base image was incompatible.

# How

As of https://github.com/racker/salus-app-base/pull/13 the base image is Java 11 and non-Alpine, so that default will work well across all of our applications now.

## How to test

Do a local docker build and confirm the container is able to startup properly (ignoring application level connectivity issues):

```
export SHORT_SHA=HEAD
mvn package jib:dockerBuild -DskipTests=true
docker run -it --rm racker/salus-telemetry-presence-monitor
```
